### PR TITLE
Update tracks and add slack channel.

### PR DESCRIPTION
**Pull request by voidtek**

### CODE REVIEW

File: docs/openeuropa-components.md [hunk 1]
```diff
@@ -16,71 +16,69 @@ Below the main, active tracks maintained by the OpenEuropa core team:
...
 ### Editorial track
 
-Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)
-  
-Components:
 - [OpenEuropa Editorial](https://github.com/openeuropa/oe_editorial)
 
-### Multilingual track
+Maintained by [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)\
+Slack Channel [#openeuropa-editorial](http://platform-ec-europa.slack.com/app_redirect?channel=openeuropa-editorial)
```
Comment (richardcanoe) [inline docs/openeuropa-components.md]: Why put a link to a private Slack in a public document?

Comment (voidtek) [reply docs/openeuropa-components.md]: Good question.
We could add the information that we should have a `@ext.ec.europa.eu` email address to create an account.
image

Comment (robstoll) [reply docs/openeuropa-components.md]: why not use something which is open-source instead ;)

**Comment by ademarco** [comment]
Closing as obsolete.

**Pull request closed without merging**